lint.sh

```diff
@@ -2,4 +2,6 @@
 
 set -eou pipefail
 
-./tools/bazel run //tools:buildifier@fix
+bazel run //tools:buildifier@fix
```

Comment:
Is there really a need for two commands? Why not have one with lint and format?

Reply:
moved to a single command